chukwuemekachm/Population-Management-System
# #165071678 Completes the create location mutation endpoint

#### What does this PR do?
- This PR completes the `createLocation` mutation to enable authorized users to create new locations on the platform.
#### Description of Task to be completed?
- add type definitions for new objects
- add create location resolver
- create permissions middleware to protect secure endpoints
- add integration tests
#### How should this be manually tested?
- Set up the project locally
- Open your browser and create a `createLocation` mutation on this endpoint `localhost:<PORT>/graphql`
- Observe the behaviour
#### Any background context you want to provide?
- N/A
#### What are the relevant pivotal tracker stories?
- [#165071678](https://www.pivotaltracker.com/story/show/165071678)
#### Screenshots (if appropriate)

<img width="1440" alt="Screenshot 2019-04-02 at 9 15 33 PM" src="https://user-images.githubusercontent.com/33798252/55434298-3470c200-558f-11e9-8aca-45e88124121e.png">


<img width="1440" alt="Screenshot 2019-04-02 at 9 15 00 PM" src="https://user-images.githubusercontent.com/33798252/55434300-35095880-558f-11e9-973e-499d5d594806.png">


#### Questions:
- N/A
